packages/block-editor/src/components/button-block-appender/style.scss

```diff
@@ -0,0 +1,34 @@
+
+.button-block-appender {
```

Comment:
This does not adhere to naming conventions:

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#naming

It should be `.block-editor-button-block-appender`

Reply:
>This does not adhere to naming conventions:

In mentioning this, I note that `block-list-appender` is wrong too, but apparently already existed this way, so there's no changing it now.

Reply:
👍 should be fixed.